test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
+
+ENV password=notpassword1
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 2aac5b31-cb47-4dca-a435-99068e0d59f2 -->